contentcuration/contentcuration/assets/channelList/vuex/channelList/mutations.js

```diff
@@ -0,0 +1,73 @@
+import Vue from 'vue';
+import { prepChannel, setDirtyTracking } from '../../utils';
+
+/* CHANNEL LIST MUTATIONS */
+function mergeChannel(channelsMap, channel) {
```

Comment:
This function ensures that we persist any previously existing channel data in the store - but any updated values are overwritten.